### ISSUE TITLE
Copy directly to bundle format?

### ISSUE THREAD

**Issue by jwendell** [containers/skopeo#342]
Currently if we want to run runc on top of a downloaded image we must take 2 steps:
1. download the image with skopeo
`skopeo copy docker://busybox oci:busybox-oci`
2. convert the image to a bundle format
`mkdir busybox-bundle`
`oci-image-tool unpack  busybox-oci busybox-bundle`

What if skopeo could directly copy to a bundle format? Something like
`skopeo copy docker://busybox oci-bundle:busybox-bundle`

Does it make sense?

**Comment by runcom** [comment]
Well, it does as something handy to have but we do have many tiny applications so that you can construct your flow as you wish (you can also use `umoci` to unpack images).  That said, again, this does make sense to me if someone is willing to work on it :)

**Comment by rhatdan** [comment]
@jwendell @runcom @mtrmac @vrothberg Is this still something of interest or is what podman does enough to satisfy the need?

**Comment by jwendell** [comment]
I don't care anymore, feel free to close this :)

**Comment by mtrmac** [comment]
Thanks, closing.

(I would expect `podman pull`&`podman run` to be the convenient path for most users. Specialist uses of `runc` directly may emerge again, we can reopen then.)

**Comment by DamienRobert** [comment]
I would have liked this feature to test `systemd-nspawn --oci-bundle=`. `oci-image-tool` / `umoci` do the job, but it would be nice if this was integrated to `skopeo`.

**Comment by DaanDeMeyer** [comment]
Agreed, having to install both skopeo and umoci to get oci bundles that can be run using systemd-nspawn feels a bit overkill. Can this be reopened?

**Comment by vrothberg** [comment]
Sure, we can re-open. Personally, I think that skopeo+umoci do the job well together, so the use-case is covered and widely used. However, I have no objections against an `oci-bundle` transport.

Contributions are always welcome!

**Comment by c-goes** [comment]
These two tools are often used to togther, for instance LXC uses both Skopeo and Umoci to create containers from OCI (https://github.com/lxc/lxc/blob/master/templates/lxc-oci.in)
However, conversion to oci-bundle in Skopeo makes sense because Podman has the `--rootfs` flag to run container from rootfs. You can use Skopeo/Podman to replace `runc` and have all the advantages of Podman without having to worry about Overlayfs. (I don't want to use Overlayfs and still want all features from Podman).
